### PR TITLE
Add example of Java manual instrumentation with the agent

### PR DESCRIPTION
**Pull request by malafeev**

### CODE REVIEW

File: examples/agent-manual/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    1. Any method can be annotated with `@WithSpan` annotation to create span with duration of method call
+        ```java
+        @WithSpan
+        public void method() {
+          // ...
+        }
+        ```
+
+    1. Manually get tracer and create span
```
Comment (kayousterhout) [inline examples/agent-manual/README.md:39]: is it intentional that all of the numbers here are "1." ?  Does markdown re-number for you?

Comment (kayousterhout) [reply examples/agent-manual/README.md:39]: nvm I just went and looked in your branch and saw that this is what happens! 👍

**Comment by kayousterhout** [comment]
Does it make sense / work to put this in the agent folder instead, so it's all one example?  That seems like it might be easier for folks to find and avoids us having another directory of examples to maintain.

**Comment by carlosalberto** [comment]
> so it's all one example?

Sounds like a good idea to me. Let's just mention that the `@WithSpan` usage is **optional** (i.e. users don't think they *have to* use it).

**Comment by malafeev** [comment]
@carlosalberto @kayousterhout I created separate example to show that I added explicit dependencies in the pom.xml on opentelemetry api. 
But for agent example user should not modify existing application.

All in one would confuse user to add opentelemetry  jars to pom.xml even for auto instrumentation only.

**Comment by carlosalberto** [comment]
> But for agent example user should not modify existing application. All in one would confuse user to add opentelemetry jars to pom.xml even for auto instrumentation only.

Good point. Maybe we can also add a comment in the pom file to mention also that such dependency is optional, etc. But overall, based on your feedback, I'm fine either way. Maybe @kayousterhout can untie the issue? ;)

**Comment by kayousterhout** [comment]
Sorry for the slow reply - I'd do @carlosalberto's suggestion: combine the examples but leave a clear comment in the pom file that's something like "This dependency is only need to add manual instrumentation; for auto-instrumentation only, it can be omitted."  Part of the motivation here is that we have so many example repositories that it's hard to maintain them all, and then they get out of date, and then they stop being useful, so the hope is to try to combine them into as few as possible.

**Comment by malafeev** [comment]
merged two examples into one

**Review by carlosalberto — APPROVED** [review]

**Pull request merged**